akka-edge-docs/src/main/paradox/guide/5-charging-station.md

```diff
@@ -0,0 +1,264 @@
+we need to combine all such sources and destinations into single gRPC services:
+
+Scala
+:  @@snip [Main.scala](/samples/grpc/restaurant-drone-deliveries-service-scala/src/main/scala/central/Main.scala) { #replicationEndpoint }
+
+Java
+:  @@snip [Main.java](/samples/grpc/restaurant-drone-deliveries-service-java/src/main/java/central/Main.java) { #replicationEndpoint }
+
+For the local-drone-control service we also create `ReplicationSettings` but pass them to a separate initialization method `Replication.grpcEdgeReplication`.
+Since the edge node will be responsible for creating connections, no gRPC services needs to be bound: 
+
+Scala
+:  @@snip [ChargingStation.scala](/samples/grpc/local-drone-control-scala/src/main/scala/charging/ChargingStation.scala) { #edgeReplicaInit }
+
+Java
+:  @@snip [ChargingStation.java](/samples/grpc/local-drone-control-java/src/main/java/charging/ChargingStation.java) { #edgeReplicaInit }
+
+The returned `EdgeReplication` instance gives us access to a `entityRefFactory` for sending messages to the charging stations.
+
+
+## Service for interacting with the charging station
+
+In the restaurant-drone-deliveries service we introduce a separate gRPC endpoint for creating and looking at charging 
+station state:
+
+Scala
+:  @@snip [charging_station_api.proto](/samples/grpc/restaurant-drone-deliveries-service-java/src/main/protobuf/charging/charging_station_api.proto) { }
+
+Java
+:  @@snip [charging_station_api.proto](/samples/grpc/restaurant-drone-deliveries-service-scala/src/main/protobuf/charging/charging_station_api.proto) { }
+
+The service implementation takes the `entityRefFactory` as a constructor parameter and uses that to create `EntityRef` instances
+for specific charging stations to interact with them:
+
+Scala
+:  @@snip [ChargingStationServiceImpl.scala](/samples/grpc/restaurant-drone-deliveries-service-scala/src/main/scala/charging/ChargingStationServiceImpl.scala) { }
+
+Java
+:  @@snip [ChargingStationServiceImpl.java](/samples/grpc/restaurant-drone-deliveries-service-java/src/main/java/charging/ChargingStationServiceImpl.java) { }
+
+## Interacting with the charging station at the edge
+
+The local-drone-control service does not contain the full gRPC API for interaction but instead has a single `goCharge` method
+in the drone gRPC service:
+
+Scala
+:  @@snip [DroneServiceImpl.scala](/samples/grpc/local-drone-control-scala/src/main/scala/local/drones/DroneServiceImpl.scala) { #charging }
+
+Java
+:  @@snip [DroneServiceImpl.java](/samples/grpc/local-drone-control-java/src/main/java/local/drones/DroneServiceImpl.java) { #charging }
+
+
+## Running
+
+The complete sample can be downloaded from GitHub, but note that it also includes the next steps of the guide:
+
+* Java: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-java
+* Scala: https://github.com/akka/akka-projection/tree/main/samples/grpc/restaurant-drone-deliveries-service-scala
```

Comment:
These should be links to the zip files. I see that we still have a few github links remaining. Must be an oversight, can you change all of them to the zip files.